cmd/warpforge/main_test.go

```diff
@@ -82,12 +78,30 @@ func cleanRunRecord(str string) string {
 	return strings.TrimSpace(str)
 }
 
-func execFn(args []string, stdin io.Reader, stdout, stderr io.Writer) (int, error) {
-	err := makeApp(stdin, stdout, stderr).Run(args)
-	if err != nil {
-		return 1, err
+func buildExecFn(projPath string) func(args []string, stdin io.Reader, stdout io.Writer, stderr io.Writer) (int, error) {
```

Comment:
I have a suspicion that having a closure factory like this is gonna make life worse when we try to get error analysis piped through here properly (since I ran into that problem in my PR last week).

Fair enough to leave that to fix when we come to it, though.

Reply:
Yeah.... you're probably not wrong but also seems like a future problem.